pandas/core/frame.py

```diff
@@ -664,17 +672,29 @@ def itertuples(self, index=True):
         b     2   0.2
         >>> for row in df.itertuples():
         ...     print(row)
-        ('a', 1, 0.10000000000000001)
-        ('b', 2, 0.20000000000000001)
+        ...
+        Pandas(Index='a', col1=1, col2=0.10000000000000001)
+        Pandas(Index='b', col1=2, col2=0.20000000000000001)
 
         """
         arrays = []
         if index:
             arrays.append(self.index)
+            fields = ["Index"] + list(self.columns)
+        else:
+            fields = self.columns
 
         # use integer indexing because of possible duplicate column names
         arrays.extend(self.iloc[:, k] for k in range(len(self.columns)))
```

Comment:
Do you mean something along the lines of 

```python
        arrays = []
        fields = []                                                     
        if index:                                                               
            arrays.append(self.index)                                           
            fields.append("Index")
                                                                                
        # use integer indexing because of possible duplicate column names       
        arrays.extend(self.iloc[:, k] for k in range(len(self.columns)))        
                                                                                
        # `rename` is unsupported in Python 2.6                                 
        try:                                                                    
            itertuple = collections.namedtuple(name, fields+list(self.columns), rename=True)       
        except:                                                                 
            # fallback to regular tuples                                        
            return zip(*arrays)                                                 
                                                                                
        return (itertuple(*row) for row in zip(*arrays)) 
```